styleguide/screens/Patterns/Cards/Cards.js

```diff
@@ -0,0 +1,656 @@
+                    alt: 'hello2',
+                  }]
+                }
+                href="#"
+              />
+            ` }
+          >
+            <div className={ css.cardContainer }>
+              <SpaceListingCard
+                price="$10,000,000"
+                priceUnit="/day"
+                location="Shoreditch, London in the Greater London Area"
+                size="1000 sqft"
+                name="A really long Bold Street Shop, maybe the biggest shop you've ever seen"
+                images={
+                  [{
+                    src: 'https://source.unsplash.com/random/500x500',
+                    alt: 'hello',
+                  }, {
+                    src: 'https://source.unsplash.com/random/500x503',
+                    alt: 'hello2',
+                  }, {
+                    src: 'https://source.unsplash.com/random/500x502',
+                    alt: 'hello',
+                  }, {
+                    src: 'https://source.unsplash.com/random/500x501',
+                    alt: 'hello2',
+                  }]
+                }
+                href="#"
+                className={ css.card }
+              />
+            </div>
+          </Specimen>
+        </D>
+        <D>
+          <H level={ 2 } className={ shared.componentTitle }>Space Listing Card with Badge</H>
+          <T elm="p" className={ m.mtr }>
+            Should be used to display additional information to a Space Listing Card (i.e NEW).
+            Using the <C>Badge</C> component.
+          </T>
+          <Specimen
+            classNames={ {
+              root: m.mtr,
+              specimenContainer: m.par,
+            } }
+            code={ dedent`
+              <SpaceListingCard
+                price="$10,000,00000000000000000000000"
+                priceUnit="/day"
+                badge={ <Badge context="primary" hollow={ false }>New</Badge> }
+                location="Shoreditch, London in the Greater London Area"
+                size="1000 sqft"
+                name="A really long Bold Street Shop, maybe the biggest shop you've ever seen"
+                images={
+                  [{
+                    src: 'https://source.unsplash.com/random/500x500',
+                    alt: 'hello',
+                  }, {
+                    src: 'https://source.unsplash.com/random/500x503',
+                    alt: 'hello2',
+                  }, {
+                    src: 'https://source.unsplash.com/random/500x502',
+                    alt: 'hello',
+                  }, {
+                    src: 'https://source.unsplash.com/random/500x501',
+                    alt: 'hello2',
+                  }]
+                }
+                href="#"
+              />
+            ` }
+          >
+            <div className={ css.cardContainer }>
+              <SpaceListingCard
+                price="$10,000,00000000000000000000000"
```

Comment:
I think we should put more sensible content in the style guide. This was fine in the storybook when it was private, but I don't think we should put unrealistic values now